### PR TITLE
[Icons]Revision of TechDraw Icons(Hatch)

### PR DESCRIPTION
**Pull request by Reqrefusion**
#14145

### CODE REVIEW

File: src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_GeometricHatch.svg
Comment (kadet1090) [inline src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_GeometricHatch.svg:1]: @WandererFan Is this icon ok?

Comment (maxwxyz) [reply src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_GeometricHatch.svg:1]: the .pat is not visible in the toolbar for me.
What about using a different hatch style than the other icon and just write PAT in the part like in the export as SVG icon?

**Review by maxwxyz — CHANGES_REQUESTED** [review]
One minor change

File: src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_GeometricHatch.svg
Comment (kadet1090) [reply src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_GeometricHatch.svg:1]: pat in the shape should work fine and as it has known background we can make it strokeless

**Comment by Reqrefusion** [comment]
@chennes It can be merged. It needs to be merged today.

File: src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_GeometricHatch.svg
Comment (Reqrefusion) [reply src/Mod/TechDraw/Gui/Resources/icons/actions/TechDraw_GeometricHatch.svg:1]: I followed Kadet's suggestion. I think it is a development that the important thing is to point out that it is different. I don't understand anything in the old icon.

**Pull request merged**